frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -6,56 +6,57 @@ import {
   getPVCSourceByDisk,
 } from '../../selectors/vm/selectors';
 import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
-import { VMKind } from '../../types';
-import { V1Disk } from '../../types/vm/disk/V1Disk';
+import { VMKind, BootableDeviceType } from '../../types';
+import { deviceLabel } from '../boot-order/constants';
```

Comment:
can we move this function out of the boot-order if it is used in multiple places?